### ISSUE TITLE
move subdomain to domain

### ISSUE THREAD

**Issue by dinospla** [PrestaShop/PrestaShop#18043]
#### Describe the bug

hi
i move sudomain to domain and i have this issue!

backoffice
![backoffice](https://user-images.githubusercontent.com/61296901/76169513-a4190500-6181-11ea-893c-1f2a4f7c35ef.jpg)

frontoffice
![frontoffice](https://user-images.githubusercontent.com/61296901/76169516-a8452280-6181-11ea-980a-ccc60fbbebc4.jpg)

#### Expected behavior

Explain what you expected to happen instead.

i change 
shop_uri
shop_domain
shop_domain_ssl
.htaccess

 to domain !

#### Additional information

* PrestaShop version: 1.7.6.1
* PHP version: 7.033

**Comment by khouloudbelguith** [comment]
Hi @dinospla,

I moved my Project to a new domain without errors.
1. I made a backup for my files & my database
2. I uploaded my store's files to your new server
3. I imported the old store's database to the new server
4. In file /app/config/parameters.php, I configured my PrestaShop site.
4.1. DB_SERVER: database's server;
4.2. DB_NAME: Name of the database from step 5;
4.3. DB_USER: User associated with the database;
4.4. DB_PASSWD: The user's password;
5. in the Table ps_shop_url, I changed with the 
5.1. domain => new domain
5.2. domain_ssl => new domain_ssl
5.3. physical_uri => new physical_uri
6. I cleared the cache manually by deleting all the folder into var/cache (prod & dev).

=> OK => no problems.

Thanks to check & feedback.
